hardware/camera/hqcam_filter_removal.md

```diff
@@ -2,15 +2,21 @@
 
 The High Quality Camera contains an IR filter, which is used to reduce the camera’s sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
 
-**This procedure cannot be reversed:** the adhesive that attaches the filter will not survive being lifted and replaced, and the IR filter may crack when it is removed. Removing it will void the warranty on the product. Nevertheless, removing the filter will be desirable to some users.
+**This procedure cannot be reversed:** the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. Removing it will void the warranty on the product. Nevertheless, removing the filter will be desirable to some users.
```

Comment:
Given @l;urch comment about not knowing the filter is attached o the lens mount, perhaps that should be mentioned?

Reply:
Maybe - or perhaps that'll be more obvious if you actually have an HQ camera in front of you?

Reply:
I've asked Simon for his input, busy as he is, because it needs someone who has a unit in front of them to refer to.